src/core/reopt.jl

```diff
@@ -387,9 +387,19 @@ function build_reopt!(m::JuMP.AbstractModel, p::REoptInputs)
         end
     end
 	
+	# Include the electric storage cost constants only if the installed_cost_constant or the replace_cost_constant is not zero
+	for b in p.s.storage.types.elec
+		if p.s.storage.attr[b].installed_cost_constant != 0 || p.s.storage.attr[b].replace_cost_constant != 0
+			@constraint(m, [b in p.s.storage.types.elec], m[:dvStorageEnergy][b] <= p.s.storage.attr[b].max_kwh * m[:dvBatteryIncluded][b]) # if the dvBatteryIncluded binary is 1, then the storage energy capacity can be greater than 0, but then the battery cost constant is also included in the costs		
+		else
+			m[:dvBatteryIncluded][b] == 0
+		end
+	end
+	
 	@expression(m, TotalStorageCapCosts, p.third_party_factor * (
 		sum( p.s.storage.attr[b].net_present_cost_per_kw * m[:dvStoragePower][b] for b in p.s.storage.types.elec) + 
```

Comment:
@zolanaj do you happen to know if this is a mistake that the `TotalStorageCapCosts` only sums across electric storage types for the power capacity and across all storage types for the energy capacity?